region/brinstar/green/Green Brinstar Main Shaft.json

```diff
@@ -4323,6 +4323,56 @@
         "Using a Super once it's on the right wall below the door can save 20 seconds."
       ]
     },
+    {
+      "link": [8, 9],
+      "name": "G-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": false
+        }
+      },
+      "requires": [
+        "Morph",
+        "h_usePowerBomb",
+        {"or": [
+          "canIBJ",
+          "h_useSpringBall",
+          "canMidAirMorph"
+        ]},
+        "h_bombThings",
+        "h_usePowerBomb",
+        "canBeVeryPatient",
+        "h_CrystalFlashForReserveEnergy",
+        {"or": [
+          "canConsecutiveWalljump",
+          "canLongIBJ",
+          "SpaceJump"
+        ]},
+        {"or": [
+          {"canShineCharge": {"usedTiles": 17, "openEnd": 1}},
+          {"and": [
+            "h_frozenEnemyRunway",
```

Comment:
Frozen runway and LongIBJ look like they add another lap of the zeela around the room.
Same with damaging down in the worst case.

Reply:
Mid-air CF didn't pan out - there's just nowhere with enough space, and it takes as many as 3 laps to damage down from 1499 energy, so that's how many Supers it'll take if E-Tanks can't be disabled.